index.html

```diff
@@ -196,6 +196,39 @@ <h2 class="myself">About me</h2>
        </div>
     </article>
    </section>
+
+   <section id="contact" class="site-section section-form text-center">
+    <div class="contain">
+      <p id="paragraph2">I'm always <br>interested in hearing about new projects, <br>so if you'd like to chat please get in touch.</p>
+       <br><br>
+     <form class="myform">
+       <div class="row">
+        <div class="col-sm-6">
+          <input type="text" name="name" class="form-control" placeholder="Full Name" required>
+        </div>
+        <div class="col-sm-6">
+          <input type="email" name="email" class="form-control" placeholder="Email Address" required>
+         </div>
+         <div class="col-sm-6">
+            <textarea name="message" id="message" class="form-control" placeholder="Enter text here" required></textarea>
+         </div>
```

Comment:
#### Dear @Ellyboi, to make your form value secure we need to add some HTML5 validations. 
- [ ] Name input should have a `maxlength="30"` .
- [ ] Message input should have `maxlength="500"`